examples/process.rs

```diff
@@ -43,6 +43,7 @@ async fn flip_the_table(p: process::Process) -> process::ProcessResult<prettytab
     table.add_row(row!["PID", p.pid()]);
     table.add_row(row!["Parent PID", p.parent_pid().await?]);
     table.add_row(row!["Name", p.name().await?]);
+    table.add_row(row!["User", p.user().await?.username()]);
```

Comment:
As it is not implemented yet for other OSes, it will break CI and the example itself
```suggestion
    #[cfg(target_os = "windows")]
    table.add_row(row!["User", p.user().await?.username()]);
```